UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -162,7 +162,7 @@ public void Heartbeat(float efficiency)
 				implant.BloodPumpedEvent(BloodToGive);
 			}
 			if (RelatedPart.HealthMaster.IsDead) return; //For some reason the heart will randomly still continue to try and beat after death.
-			if (RelatedPart.BloodContainer.CurrentReagentMix.MajorMixReagent == salt || RelatedPart.BloodContainer.AmountOfReagent(salt) > danagerSalt)
+			if (RelatedPart.BloodContainer.CurrentReagentMix.MajorMixReagent == salt || RelatedPart.BloodContainer.AmountOfReagent(salt) * 100 > danagerSaltLevel)
 			{
 				Chat.AddActionMsgToChat(RelatedPart.HealthMaster.gameObject,
 					"<color=red>You hold your chest as you feel your heart giving up!</color>",
```

Comment:
might be worth referencing because of the salt?
something like 
at the end " too much salt!"

Reply:
in the code or message?

Reply:
message

Reply:
So, if the player gets poisoned with too much salt they know why

Reply:
I think that saying "too much salt" looks bad so instead I made the message more unique to reflect the effect of salt poisoning